### ISSUE TITLE
Not installing on 6.04.18165

### ISSUE THREAD

**Issue by yelmarb** [elgatosf/streamdeck-cpu#23]
Would love to install CPU on my stream deck but it's not showing up anywhere.

**Comment by rgarcia525** [comment]
Hey @yelmarb  - try updating. I had the CPU thing installed for a year+ but noticed yesterday that it stopped working for me.

So I updated this morning and it works fine now!

_See screenshot below_
1. Click the settings cog wheel
2. Click 'Check for Updates'

I hope that helps!

<img width="832" alt="stream deck update" src="https://user-images.githubusercontent.com/11171271/220660756-13f865ac-7d79-4d4b-9d54-368bf0c9e473.png">

**Comment by yelmarb** [comment]
Thanks @rgarcia525 I've updated the Stream Deck and while it "installs" it doesn't show up anywhere...

**Comment by ChekTek** [comment]
I'm not able to reproduce this issue. Please provide your operating system information and Stream Deck version. If you navigate to the store page from within Stream Deck, does the store display the "uninstall" button?

**Comment by yelmarb** [comment]
Yes there is an uninstall button, so I've tried uninstalling and reinstalling but it doesn't show up. Stream Deck is the latest version 6.0.5 and I'm running Windows 10 Pro 64 bit version 22H2.
![Screenshot 2023-02-23 101549](https://user-images.githubusercontent.com/125245265/220785150-20734a2c-ba1e-4176-a493-8c859d1ff653.jpg)
